### ISSUE TITLE
Add support to provide a password for a user

### ISSUE THREAD

**Issue by rsakmyster** [Axway-API-Management-Plus/apim-cli#172]
**API-Manager CLI Version**
1.3.5

**API-Manager and Service-Pack Version**

**Question**
Does the user import cli support password specification for the user (json element)?  This would be very useful when bulk loading users to give them an initial password.

**What I've tried so far**
I see no references in docs.  However, I have tried password and password1/2 json elements.  No effect

**Additional information**

**Comment by cwiechmann** [comment]
Hi @rsakmyster, Thanks for reporting this issue. Unfortunately that is actually not supported, but it definitely should. The feature will be added with the next release.

**Comment by cwiechmann** [comment]
Support for passwords has been added and will become available with version 1.3.8. With that version you can set a password for a user:
```json
{
  "loginName": "myuser",
  "password": "password4user",
  "name": "Sample user",
  "organization": "API Development"
}
```

**The password will only be set, if the user is created.** During the update of an existing user the following message is logged, when a password is given:
`Password of existing user: 'myuser' (76eb1b06-c067-4347-81f2-2b31b97e773c) will not be updated.`

To change the password of an existing users you can use a new command:
`apim user changepassword Changes the password of the selected users.`

Is this sufficient for your use-case @rsakmyster? 

Question: Do you consider the command resetpassword and enable/disable users are helpful/important to have?

**Comment by rsakmyster** [comment]
That sounds great.  When is that version available?

Rob  Sakmyster
Peraton
TALOS MARS Integration Lead
703-561-3715

From: Chris Wiechmann ***@***.***>
Sent: Friday, May 28, 2021 7:36 AM
To: Axway-API-Management-Plus/apim-cli ***@***.***>
Cc: Sakmyster, Robert J [US] (PN) ***@***.***>; Mention ***@***.***>
Subject: EXT :Re: [Axway-API-Management-Plus/apim-cli] Add support to provide a password for a user (#172)


Support for passwords has been added and will become available with version 1.3.8. With that version you can set a password for a user:

{

  "loginName": "myuser",

  "password": "password4user",

  "name": "Sample user",

  "organization": "API Development"

}

The password will only be set, if the user is created. During the update of an existing user the following message is logged, when a password is given:
Password of existing user: 'myuser' (76eb1b06-c067-4347-81f2-2b31b97e773c) will not be updated.

To change the password of an existing users you can use a new command:
apim user changepassword Changes the password of the selected users.

Is this sufficient for your use-case @rsakmyster<https://github.com/rsakmyster>?

Question: Do you consider the command resetpassword and enable/disable users are helpful/important to have?

**Comment by cwiechmann** [comment]
I'll release a new version maybe next week. Watch the GitHub repository.

Any thoughts on the other question concerning user management? @rsakmyster
